install/kubernetes/helm/subcharts/tracing/templates/deployment-zipkin.yaml

```diff
@@ -0,0 +1,57 @@
+{{ if eq .Values.provider "zipkin" }}
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-zipkin
+  labels:
+    app: zipkin
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  template:
+    metadata:
+      labels:
+        app: zipkin
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.zipkin.hub }}:{{ .Values.zipkin.tag }}"
+          ports:
+            - containerPort: {{ .Values.zipkin.queryPort }}
+          livenessProbe:
+            initialDelaySeconds: {{ .Values.zipkin.probeStartupDelay }}
+            tcpSocket:
+              port: {{ .Values.zipkin.queryPort }}
+          readinessProbe:
+            initialDelaySeconds: {{ .Values.zipkin.probeStartupDelay }}
+            httpGet:
+              path: /health
+              port: {{ .Values.zipkin.queryPort }}
+          resources:
+            limits:
```

Comment:
Resources should use the same approach as in other deployments, e.g. [prometheus](https://github.com/istio/istio/blob/master/install/kubernetes/helm/subcharts/prometheus/templates/deployment.yaml#L49-L53), [jaeger](https://github.com/istio/istio/blob/master/install/kubernetes/helm/subcharts/prometheus/templates/deployment.yaml#L49-L53), ...

Same applies for [priorityClassName, imagePullSecrets](https://github.com/istio/istio/blob/master/install/kubernetes/helm/subcharts/tracing/templates/deployment.yaml#L27-L35) and [affinity](https://github.com/istio/istio/blob/master/install/kubernetes/helm/subcharts/tracing/templates/deployment.yaml#L77-L78).

Reply:
Done. I used separate `zipkin` and `jaeger` resource blocks, however, as I feel like they may not need the same default resources and I want to ease switching between the two.